src/pretalx/person/models/profile.py

```diff
@@ -30,6 +30,7 @@ class SpeakerProfile(LogMixin, models.Model):
 
     class urls(EventUrls):
         public = '{self.event.urls.base}/speaker/{self.user.code}'
+        talks_ical = '{self.urls.public}/talks.ical'
```

Comment:
I suppose it would make more sense to put the speaker code or speaker name here, in addition. Otherwise you'll end up with talks.ical, talks (2).ical, etc.

Also, we use .ics in other places, if memory serves, so I'd prefer this for consistency's sake.

Reply:
The downloaded filename contains both the slug and a (cleaned) speaker name: https://github.com/pretalx/pretalx/pull/304/files#diff-3c2419007a707ff9150a067e5d3c3b57R62

Reply:
ical => ics  fixed. :)